docs/data/components/accordion/accordion.mdx

```diff
@@ -0,0 +1,332 @@
+---
+productId: base-ui
+title: React Accordion components
+components: AccordionRoot, AccordionItem, AccordionHeader, AccordionTrigger, AccordionPanel
+githubLabel: 'component: accordion'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/accordion/
+packageName: '@base_ui/react'
+---
+
+# Accordion
+
+<Description />
+
+<ComponentLinkHeader design={false} />
+
+## Introduction
+
+<Demo demo="UnstyledAccordionIntroduction" defaultCodeOpen="false" bg="gradient" />
+
+## Installation
+
+<InstallationInstructions componentName="Accordion" />
+
+## Anatomy
+
+Accordions are implemented using a collection of related components:
+
+- `<Accordion.Root />` is a top-level component that wraps the other components.
+- `<Accordion.Item />` is a component that wraps each section of content and it's associated `Trigger`
+- `<Accordion.Trigger />` is a button that toggles the open state of it's associated `Item`
+- `<Accordion.Header />` is a heading (`h3` by default) that wraps the `Trigger`
+- `<Accordion.Panel />` is the element that contains content in a `Item`
+
+```tsx
+<Accordion.Root>
+  <Accordion.Item>
+    <Accordion.Header>
+      <Accordion.Trigger>Toggle one</Accordion.Trigger>
+    </Accordion.Header>
+    <Accordion.Panel>Panel one content</Accordion.Panel>
+  </Accordion.Item>
+  <Accordion.Item>
+    <Accordion.Header>
+      <Accordion.Trigger>Toggle two</Accordion.Trigger>
+    </Accordion.Header>
+    <Accordion.Panel>Panel two content</Accordion.Panel>
+  </Accordion.Item>
+</Accordion.Root>
+```
+
+## Value
+
+Each `Accordion.Item` is represented by a value, which by default is its zero-based index by DOM position.
+The first `Item` has an implicit `value` of `0`, the second one `Item` a `value` of `1`, and so on.
+
+The open state of the accordion is represented an array holding the `value`s of all open `Item`s.
+
+You can optionally specify a custom `value` prop on `Item`:
+
+```tsx
+<Accordion.Root>
+  <Accordion.Item value="one">
+    <Accordion.Header>
+      <Accordion.Trigger>Toggle one</Accordion.Trigger>
+    </Accordion.Header>
+    <Accordion.Panel>Panel one content</Accordion.Panel>
+  </Accordion.Item>
+  <Accordion.Item value="two">
+    <Accordion.Header>
+      <Accordion.Trigger>Toggle two</Accordion.Trigger>
+    </Accordion.Header>
+    <Accordion.Panel>Panel two content</Accordion.Panel>
+  </Accordion.Item>
+</Accordion.Root>
+```
+
+### Default value
+
+When uncontrolled, use the `defaultValue` prop to set the initial state of the accordion:
+
+```tsx
+<Accordion.Root defaultValue={[0]}>
```

Comment:
I agree the API for Tabs and Accordion should be as similar as possible, wonder what @vladmoroz and @colmtuite think?
The API would be `openPanels`/`defaultOpenPanels`? and `onOpenPanelChange`?

Reply:
@michaldudak what would `value` become on the item itself? E.g. here `<Accordion.Item value="1">`

Reply:
`value` on items still makes sense to me.

Reply:
If items have `value`, I think the connection gets lost then if the root uses anything else other than `value`

Reply:
For using `id`, I worry it's not a good idea to conflate the "panel value" with the `id` attribute 🤔 
but since this is equally relevant to `Tab`s let's discuss this in the next call to unblock this PR 🙏

Reply:
I'm not entirely convinced it's the best idea either, especially that on Tabs you can use non-string values as well, which won't be possible with `id`. But then, is it a real use case?

Reply:
We discussed this and decided to stay with `value`/`onValueChange` for general consistency, all the other options we could come up with are a bit clumsy by comparison